staging/src/k8s.io/apimachinery/pkg/util/intstr/intstr_test.go

```diff
@@ -21,9 +21,20 @@ import (
 	"reflect"
 	"testing"
 
+	"github.com/gogo/protobuf/proto"
+
 	"sigs.k8s.io/yaml"
 )
 
+func TestIntstr(t *testing.T) {
+	x := FromString("1")
+	y := FromString("2")
+	proto.Merge(&x, &y)
```

Comment:
Do we generally support proto.Merge? What are people using this for? What is the result of this operation?

Reply:
We are using `proto.Clone` which seems to call proto.Merge at some point in the call tree

Reply:
Why are you using that rather than the DeepCopy functions we generate? (assuming it does the same thing?)